src/stache/includes/cli/lint-fix.html

```diff
@@ -0,0 +1 @@
+<stache-code>--fix</stache-code> — Attempts to automatically "fix" any linting errors found.  The default is false.  Please use caution when setting this to true, as it will overwrite files. This flag is passed directly <stache-code>tslint</stache-code> library.
```

Comment:
What does "fix" mean? If it's really fixing it, I vote to remove the quotation marks so it's clearer. Also, is this a property or option? Wasn't sure how to reference it in the edits below.  

Attempts to automatically fix linting errors. By default, this property (option?) is set to <stache-code>false</stache-code>. Use caution when setting the value to true as it overwrites files. This flag is passed directly <stache-code>tslint</stache-code> library.

Reply:
Very good points.  @blackbaud-johnly had some very similar verbal feedback.  I believe he's going to submit a PR in to mine so we can make sure to get that addressed.

Reply:
Addressed in separate PR